test/ui/cypress/integration/serving.spec.js

```diff
@@ -12,13 +12,7 @@ describe('OCP UI for Serverless', () => {
     describe('with authenticated via Web Console', () => {
       openshiftConsole.login()
     })
-    describe('remove kservice', () => {
-      showcaseKsvc.removeApp()
-    })
-  })
-
-  afterEach(() => {
-    describe('remove kservice', () => {
+    describe('remove app', () => {
```

Comment:
In `ShowcaseKservice` class what is the benefit of using `app` instead of `service` for testing?

Reply:
The OpenShift application is just a grouping of resources in ODC user interface. It relies on using the `app.kubernetes.io/part-of=XXX` label, and presents itself as a logical boundary in the UI.

It also allows the user to delete all the elements of the application at once, using the ODC UI. In this case, I'm using it because by creating a Kservice via ODC form, an _ImageStream_ resource is created as well. To make sure it would be deleted, I'm utilizing the app mechanism.

Reply:
Thanks for the explanation I was wondering if the odc app is the same thing here.